src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -136,21 +136,17 @@ def prepare_example_inputs(inputs, input_signature):
             if inputs is not None:
                 if isinstance(inputs, dict):
                     if input_signature is not None:
-                        ordered_inputs = []
                         used_sign = []
                         for key in input_signature:
                             if key not in inputs:
                                 continue
-                            ordered_inputs.append(inputs[key])
                             used_sign.append(key)
-                        inputs = ordered_inputs
                         input_signature = used_sign
-                    else:
-                        inputs = list(inputs.values())
-                        input_signature = input_signature[:len(inputs)]
+                    return {"example_kwarg_inputs": inputs}, input_signature
```

Comment:
I tried this for dolly with 3 inputs (ids, mask, past_keys_values) and it doesn't work: `jit.trace` complains about too many arguments passed (just ignoring past_key_values input for some reason in the original model signature). The only thing that works is to wrap model.forward to a separate function, define all inputs as positional arguments, and pass it in `example_inputs` instead of `example_kwarg_inputs`. I don't know why it happens, but I cannot reproduce the same issue with your test (where three inputs exist: x, y, z).

Reply:
Looks like I was able to reproduce: it works only for tensor inputs, and doesn't work for tuples. For some reason, they just cannot be passed in this way. Modified test model (`class NeuralNetwork` defined below) to expect tuple as one of the inputs (z) and pass it in `example_kwarg_inputs` to `jit.trace` and it failed with error: `Exception:
	forward() expected at most 3 argument(s) but received 4 argument(s). Declaration: forward(__torch__.___torch_mangle_1447.NeuralNetwork self, Tensor x, Tensor y) -> Tensor`.

Reply:
which pytorch version do you use?
I see in optimum code that they uses example_kwarg_input of pkv only if version 2.0.1 and on previous versions it does not work

Reply:
`torch.__version__` gives `2.0.1+cu117`

Reply:
2.0.1 looks to be the latest official version in PyPI, as you mentioned probably nightly works. OK, let's put tuple issues aside, I will cover it somehow separately.